dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/Constants.java

```diff
@@ -41,7 +41,7 @@ private Constants() {
     /**
      * alter properties
      */
-    public static final int ALERT_RPC_PORT = 50052;
+    public static final String ALERT_RPC_PORT = "alter.rpc.port";
```

Comment:
alter.rpc.port? is it `alert.rpc.port`?

Reply:
sorry，I revised it

The word "alert" in the comment is also wrong. Did I modify it by the way? Or amend it through a new issue?

Reply:
> alter.rpc.port? is it `alert.rpc.port`?

@dailidong

Reply:
you can modify it in this PR

Reply:
please modify it in this PR, thx

Reply:
> please modify it in this PR, thx

Modified to complete

Reply:
@dailidong What to do next?